src/struct/HTTP.ts

```diff
@@ -21,6 +21,128 @@ export class HTTP {
 			});
 		});
 
+		this.app.get("/imageApi", async (req, res) => {
+			const { xp, xpToLevel, level, avatarURL, tag, status } = req.query;
+			const { createCanvas, loadImage, registerFont } = require("canvas");
+			const post = {
+				nickname: tag,
+				rank: level,
+				xp: xp,
+				toLevel: xpToLevel,
+				avatar: avatarURL
+			};
+
+			const width = 840;
+			const height = 214;
+
+			const canvas = createCanvas(width, height);
+			const context = canvas.getContext("2d");
+			registerFont(__dirname + '/../assets/fonts/Poppins-Bold.ttf', { family: 'Poppins Bold' });
+			registerFont(__dirname + '/../assets/fonts/Poppins-Light.ttf', { family: 'Poppins Light' });
+			registerFont(__dirname + '/../assets/fonts/Poppins-Regular.ttf', { family: 'Poppins Regular' });
+
+			/** Background **/
+			context.fillStyle = "#010144";
+			context.fillRect(0, 0, width, height);
+			/** Background **/
+
+			/** Texts **/
+
+			context.font = "16pt 'Poppins Regular'";
+			context.textAlign = "left";
+			context.fillStyle = "#fff";
+
+			const titleText = this.client.utils.formatTitle(post.nickname);
+			context.fillText(titleText[0], 254, 145);
+			context.font = "13pt 'Poppins Light'";
+			context.textAlign = "left";
+			context.fillStyle = "#fff";
+			context.fillText(this.client.utils.formatNumber(parseInt(<string>post.xp)) + "/" + this.client.utils.formatNumber(parseInt(<string>post.toLevel)), 680, 145);
+			/** Texts **/
+
+			/** Background PINS **/
```

Comment:
What are those pins? We just need simple one-color background. If they adds some fancy stuff ı would like them to be deleted but if it looks great then no problems. Can you add a result image example to consider if these pins stays or deleted.

Reply:
it looks better than flat background :)